deploy/deploy.sh

```diff
@@ -114,32 +66,14 @@ echo "Waiting for the ${THREE_SCALE_URL} to be reachable"
 wait_for "curl -s -o /dev/null -w '%{http_code}' ${THREE_SCALE_URL} | grep 200" "3SCALE API to be reachable" "10m" "10"
 
 # Deploy the Workload App
-if [[ -z "${RHOAM}" ]]; then
-  echo "Deploying the webapp with the following parameters:"
-  echo "AMQ_ADDRESS=$AMQ_ADDRESS"
-  echo "AMQ_QUEUE=$AMQ_QUEUE"
-  echo "AMQ_CONSOLE_URL=$AMQ_CONSOLE_URL"
-  echo "RHSSO_SERVER_URL=$RHSSO_SERVER_URL"
-  echo "THREE_SCALE_URL=$THREE_SCALE_URL"
-  oc process -n $NS -f $DIR/template.yaml \
-    -p AMQ_ADDRESS=$AMQ_ADDRESS \
-    -p AMQ_QUEUE_NAME=$AMQ_QUEUE \
-    -p AMQ_CONSOLE_URL=$AMQ_CONSOLE_URL \
-    -p RHSSO_SERVER_URL=$RHSSO_SERVER_URL \
-    -p THREE_SCALE_URL=$THREE_SCALE_URL \
-    -p AMQ_CRUD_NAMESPACE=$NS \
-    -p WORKLOAD_WEB_APP_IMAGE=$IMAGE |
-    oc apply -n $NS -f -
-else
-  echo "Deploying the webapp with the following parameters:"
+echo "Deploying the webapp with the following parameters:"
   echo "RHSSO_SERVER_URL=$RHSSO_SERVER_URL"
   echo "THREE_SCALE_URL=$THREE_SCALE_URL"
   oc process -n $NS -f $DIR/template-rhoam.yaml \
     -p RHSSO_SERVER_URL=$RHSSO_SERVER_URL \
     -p THREE_SCALE_URL=$THREE_SCALE_URL \
     -p WORKLOAD_WEB_APP_IMAGE=$IMAGE |
     oc apply -n $NS -f -
```

Comment:
Very minor suggestion, I think this could be indented on the same line now it's not in the if statement.
```suggestion
echo "Deploying the webapp with the following parameters:"
echo "RHSSO_SERVER_URL=$RHSSO_SERVER_URL"
echo "THREE_SCALE_URL=$THREE_SCALE_URL"
oc process -n $NS -f $DIR/template-rhoam.yaml \
  -p RHSSO_SERVER_URL=$RHSSO_SERVER_URL \
  -p THREE_SCALE_URL=$THREE_SCALE_URL \
  -p WORKLOAD_WEB_APP_IMAGE=$IMAGE |
  oc apply -n $NS -f -
```

Reply:
That's a good point, thanks for spotting this. Updated now